modular_skyrat/code/modules/cargo/packs/medical.dm

```diff
@@ -0,0 +1,36 @@
+/datum/supply_pack/medical/firstaidmixed
+	name = "Mixed Medical Kits"
+	desc = "Contains one of each medical kits for dealing with a variety of injured crewmembers."
+	cost = 2000
+	contains = list(/obj/item/storage/firstaid/toxin,
+					/obj/item/storage/firstaid/o2,
+					/obj/item/storage/firstaid/brute,
+					/obj/item/storage/firstaid/fire,
+					/obj/item/storage/firstaid/regular)
+	crate_name = "medical kit crate"
+
+/datum/supply_pack/medical/medipens
+	name = "Epinephrine Medipens"
+	desc = "Contains two boxes of epinephrine medipens. Each box contains seven pens."
+	cost = 1300
```

Comment:
I think you might have accidentally forgotten to change the price, you got medipens as 40 credits, so it'd cost 280 to get the same amount.

Would be worth lowering the cost to somewhere between 200 and 250 since it's a bulk purchase. Alternatively, you could replace single pens with the box since only MDs are likely to bother buying them anyway and no one wants to wait around for cargo for just one medipen.

Reply:
It's more expensive because it's more convenient, and it can be bought with money from cargo, or the medical budget. The single-purchase medipens are for when people use the one they start with on someone else and wish to have a replacement without buying an entire first-aid kit (which is often actually pretty expensive for an assistant).

Also, -Crates- Need to be at least 700 credits. 700 + 560 is 1260, so i just bumped it up a bit. The pens are actually fairly more expensive in the vendors, but that's the cost of getting an immediate purchase.